apps/prairielearn/src/tests/courseEditor.test.sql

```diff
@@ -116,3 +116,10 @@ SELECT
   *
 FROM
   assessment_errors;
+
+-- BLOCK set_sharing_name
+UPDATE pl_courses
+SET
+  sharing_name = $sharing_name
+WHERE
+  id = $course_id;
```

Comment:
Why not use `updateCourseSharingName` from `src/models/course.ts`?

Reply:
done in 82215e3cec